### DISCUSSION TITLE
CSS?

### DISCUSSION THREAD

**Discussion by airbrief** [shuding/nextra#54] [Q&A] [answered]
Hi, I've just found Nextra which looks ace. How do you change the CSS? font size

**Comment by shuding** [comment, marked as answer]
You can create another CSS file and import it via _app.js.

**Reply by surjithctly** [reply]
Hey, How can I change Tailwind Classes? Currently, The design is same for every user. 

I wanted to change the look and feel completely using Tailwind. But keep all of your nice functionality. But since everything comes from a package, I'm confused. 

Please guide. 

For example, I want to change the container max width 

```css
// Change this
flex flex-1 h-full

// to this
flex flex-1 h-full max-w-2xl
```

**Reply by surjithctly** [reply]
Any thoughts on this @shuding

**Reply by shuding** [reply]
I think you can fork and create your own theme: https://github.com/shuding/nextra/tree/core/packages. There're 2 themes and you can extend your own docs theme based on the default one.

**Comment by guylepage3** [comment]
I added an `_app.js` file to the `pages` directory and added a CSS file as well.

```
import './theme.css';

export default function App({ Component, pageProps }) {
  return <Component {...pageProps} />;
}
```

This file however cascades after the `styles` in the theme.

Any suggestions? 

> — UPDATE! 
> I found a very rough solution to this here: https://github.com/shuding/nextra/discussions/54#discussioncomment-6050862

**Reply by vshwjet** [reply]
What was the solution? Can you share it here

**Comment by guylepage3** [comment]
So apparently Tailwind CSS recommends NOT to use Tailwind for any templating, design systems, etc. Since Tailwind CSS was used to create the Nextra templates, it's next to impossible to override/overwrite any of the CSS in the templates. Tailwind will need to be removed from these templates sooner rather than later or these will strictly be rendered as demo templates.

I'll open an issue and pin it here.

**Comment by guylepage3** [comment]
I randomly found some documentation on theming in one of the `Docs Theme` README.md's that helps with adding a custom css file to your docs.

https://github.com/shuding/nextra/blob/main/examples/docs/src/pages/themes/docs/index.mdx#docs-theme

```
import 'nextra-theme-docs/style.css'

export default function App({ Component, pageProps }) {
  return <Component {...pageProps} />;
}
```

**Comment by guylepage3** [comment]
Hm.. The comment marked off-topic is the "actual" answer to randomly only saying... "You can create another CSS file and import it via _app.js."

You may want to change that.

**Comment by dimaMachina** [comment]
https://nextra.site/docs/guide/custom-css
